packages/plugin-bm-api/src/graphql/resolvers/bmsBranchResolver.ts

```
@@ -0,0 +1,33 @@
+import { IContext } from '../../connectionResolver';
+import { sendCoreMessage } from '../../messageBroker';
+
+const item = {
+  async user1s(touritem: any, {}, { models, subdomain }: IContext) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)